recipe/meta.yaml

```
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: True  # [py<37 or osx]
```

```suggestion
  skip: True  # [py<37]
```
As you rebuild for `win64` only it doesn't make sense to skip `osx`